man/systemd.unit.xml

```diff
@@ -1361,6 +1361,20 @@
           </listitem>
         </varlistentry>
 
+        <varlistentry>
+          <term><varname>ConditionDMI=</varname></term>
```

Comment:
So, ConditionFirmware= already knows `device-tree-compatible(…)`, which is a parameterized test. I think we should build this in a similar way.

(Also, I think "DMI" is actually a misnomer, the standard is actually called SMBIOS, and maybe it's time to use the right wording here for new stuff).

So what about this:

```
ConditionFirmware=smbios-field(xxxx == yyy)
```

Reply:
> (Also, I think "DMI" is actually a misnomer, the standard is actually called SMBIOS, and maybe it's time to use the right wording here for new stuff).

I was struggling between DMI and SMBIOS, too. I've chosen DMI because that's was the kernel is using. But SMBIOS is also fine for me.

> So what about this:
> ```ConditionFirmware=smbios-field(xxxx == yyy)```

Sound reasonable for me. I'm going to implement this with the next update.